app/views/nics.scala.html

```diff
@@ -37,15 +37,20 @@
 
 @includes.taxs_main(title = Messages("ats.nics.tax_and_nics.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo), viewModel.utr, contentHeader = Option(headerContent), actingAttorney = actingAsAttorney) {
     @helpers.back_link(Messages("generic.back"))
-    <h1>
-        <span class="gray heading-medium block-heading">
+    @if(viewModel.isPaye){
```

Comment:
`if/else` in view which could likely be avoided with refactor

Reply:
Yep. Pls see above.